ui/src/app/app.component.ts

```diff
@@ -81,17 +90,44 @@ export class AppComponent  implements OnInit, OnDestroy {
      * SharedWorker for the others  (worker shared between tabs)
      */
     startWarningWorker(): void {
-        if (this.worker) {
-            this.stopWarningWorker();
-        }
-        this.worker = new CDSWorker('./assets/worker/web/warning.js');
-        this.worker.start({ 'user': this._authStore.getUser(), 'session': this._authStore.getSessionToken(), 'api': environment.apiURL});
-        this.worker.response().subscribe( msg => {
+        this.stopWarningWorker();
+        this.warningWorker = new CDSWorker('./assets/worker/web/warning.js');
+        this.warningWorker.start({
+            'user': this._authStore.getUser(),
+            'session': this._authStore.getSessionToken(),
+            'api': environment.apiURL});
+        this.warningWorker.response().subscribe( msg => {
             if (msg) {
                 this.zone.run(() => {
                     this._warnStore.updateWarnings(JSON.parse(msg));
                 });
             }
         });
     }
+
+    startVersionWorker(): void {
+        this.stopVersionWorker();
+        this.versionWorker = new CDSWorker('./assets/worker/web/version.js');
+        this.versionWorker.start({});
+        this.versionWorker.response().subscribe( msg => {
+            if (msg !== null) {
```

Comment:
if msg == undefined it enters in the condition. Is it the comportement desired ?